wire-ios/Wire-iOS/Sources/UserInterface/PermissionDeniedHint/NotificationAuthorizationStatusDeniedUseCases/ShouldPresentNotificationPermissionHintUseCaseProtocol.swift

```diff
@@ -0,0 +1,26 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+// sourcery: AutoMockable
+/// Determines if the user should be presented a hint, that notifications cannot be shown.
```

Comment:
I think sourcery won't pick this up if the comment isn't directly above the type declaration.

Reply:
It does, we have a few examples already in the code, like `IsSelfUserE2EICertifiedUseCaseProtocol`.